mmv1/third_party/terraform/tests/data_source_secret_manager_secret_test.go

```diff
@@ -0,0 +1,56 @@
+package google
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDataSourceSecretManagerSecret_basic(t *testing.T) {
```

Comment:
This test failed with a `no "id" found in attributes` error

Reply:
Is it similar to this? https://github.com/GoogleCloudPlatform/magic-modules/pull/4515#issuecomment-783549514